bundles/org.openhab.binding.emotiva/src/main/java/org/openhab/binding/emotiva/internal/EmotivaProcessorHandler.java

```diff
@@ -472,35 +511,40 @@ private void handleChannelUpdate(String emotivaSubscriptionName, String value, S
             try {
                 subscriptionTag = EmotivaSubscriptionTags.valueOf(emotivaSubscriptionName);
             } catch (IllegalArgumentException e) {
-                logger.debug("Property '{}' could not be mapped subscription tag, skipping", emotivaSubscriptionName);
+                logger.debug("Subscription property '{}' could not be mapped to Emotiva property tag, skipping",
+                        emotivaSubscriptionName);
                 return;
             }
 
             if (subscriptionTag.getChannel().isEmpty()) {
-                logger.debug("Subscription property '{}' does not have a corresponding channel, skipping",
+                logger.debug("Subscription property '{}' does not have a corresponding OH channel, skipping",
                         emotivaSubscriptionName);
                 return;
             }
 
-            String trimmedValue = value.trim();
-
-            logger.debug("Found subscription '{}' for '{}' and value '{}'", subscriptionTag, emotivaSubscriptionName,
-                    trimmedValue);
+            var trimmedValue = value.trim();
+            if (subscriptionTag.getChannel().equals(LAST_SEEN_STATE_NAME)) {
+                logger.trace("Subscription property '{}' with value '{}' mapped to OH channel '{}'", subscriptionTag,
+                        trimmedValue, subscriptionTag.getChannel());
+            } else {
+                logger.debug("Subscription property '{}' with value '{}' mapped to OH channel '{}'", subscriptionTag,
```

Comment:
```suggestion
                logger.trace("Subscription property '{}' with value '{}' mapped to OH channel '{}'", subscriptionTag,
```

Reply:
See other comment. I would suggest to remove the logic and log both cases to trace.